full/src/main/java/apoc/load/LoadDirectory.java

```diff
@@ -109,9 +112,11 @@
 
     // visible for test purpose
     public static String checkIfUrlBlankAndGetFileUrl(String urlDir) throws IOException {
-        return StringUtils.isBlank(urlDir)
-                ? encodePath(getDirImport())
-                : FileUtils.changeFileUrlIfImportDirectoryConstrained(urlDir.replace("?", "%3F"));
+        if (StringUtils.isBlank(urlDir)) {
+            final Path pathImport = Paths.get(getDirImport()).toAbsolutePath();
+            return chop(encodePath(pathImport.toUri().toString()));
```

Comment:
why use chop here?

Reply:
The `chop` is used to remove the final `/` from the path (`file:///My/Path/Windows/`).
It still works even without it, but it seems more correct to me.

Reply:
Would there be something already in Java we could use, rather than importing a separate dependency for it?

Reply:
Yeah, of course. Now I've changed it.